gonvme_tcp_fc.go

```diff
@@ -63,12 +67,46 @@ func (nvme *NVMeTCP) buildNVMeCommand(cmd []string) []string {
 	return command
 }
 
-// DiscoverNVMeTCPTargets - runs nvme discovery and returns a list of targets.
-func (nvme *NVMeTCP) DiscoverNVMeTCPTargets(address string, login bool) ([]NVMeTarget, error) {
+func (nvme *NVMe) getFCHostInfo() ([]FCHBAInfo, error) {
+
+	match, err := filepath.Glob("/sys/class/fc_host/host*")
+	if err != nil {
+		log.Infof("Error gathering fc hosts: %v", err)
```

Comment:
use Errorf() instead of Infof().
please fix the same in other places in this file.